demo/site/src/app/layout.tsx

```diff
@@ -1,3 +1,5 @@
+import "@comet/cms-site/css";
```

Comment:
This is breaking. However, in the next PR I copy this package, rename it to `@comet/site-next` and then restore the original cms-site package. So this change won't affect the projects